src/utils/generateRandomColor.tsx

```diff
@@ -9,4 +23,14 @@ const generateRandomColors = (numberOfColors: number) => {
   return colors;
 };
 
-export { generateRandomColor, generateRandomColors };
+const generateTerrawareRandomColors = (theme: Theme, numberOfColors: number) => {
+  const terrawareColorsList = terrawareColors(theme);
+  const colors = Array.from({ length: numberOfColors }).map((_, i) => {
+    const randomIndex = Math.floor(Math.random() * terrawareColorsList.length);
+    const selectedColor = terrawareColorsList.splice(randomIndex, 1);
+    return selectedColor[0] || generateRandomColor();
```

Comment:
if numberOfColors is bigger than terrawareColors' length, we return a random color

Reply:
We need to cycle through colors, no need to use random.
if there are 10 colors.. and 12 bars, we cycle through colors and repeat.
ie.
bar1 - color1
bar2 - color2
bar3 - color3...
bar10 - color10
bar11 - color1
bar 12 - color2

Reply:
Oh, ok, but we will have repeated colors. Also between the two chart we will get same colors for the bars

Reply:
That was my understanding from Rachel. We can check with her if needed.
In this current PR, is it possible to have 2 bars next to each other with the same color?

Reply:
It could be possible if generateRandomColor() returns same color twice, but low probability

Reply:
Ok we can get this in maybe and ask Rachel in the Jira, follow up later if needed.